libjcat/jcat-engine.c

```diff
@@ -7,14 +7,14 @@
 #include "config.h"
 
 #include "jcat-common-private.h"
-#include "jcat-context.h"
+#include "jcat-context-private.h"
 #include "jcat-engine-private.h"
 
 typedef struct {
 	JcatContext		*context;		/* weak */
 	JcatBlobKind		 kind;
 	JcatEngineVerifyKind	 verify_kind;
-	gchar			*localstatedir;
+	gboolean		 has_setup;
```

Comment:
point of preference; I think this should actually be `done_setup`

Reply:
Completely agree, good catch.